prepare_mkl.sh

```diff
@@ -115,7 +115,7 @@ if [ -z $MKLROOT ]; then
 fi
 
 # Check what MKL lib we have in MKLROOT
-if [ -z `find $MKLROOT -name libmklml_gnu.so -o -name libmklml.dylib -print -quit` ]; then
+if [ -z `find $MKLROOT -name libmklml_gnu.so -print -quit` ]; then
   USE_MKLML=0
```

Comment:
I enabled mac mkl a while back. What's wrong with this?

Reply:
Also, why is libmklml_gnu listed here while in most cases the linked library is libmklml_intel?

Reply:
This change makes USE_MKLML to 0 on linux machines.

Reply:
Not at the price of breaking mac please.

Reply:
@szha This new change from @ashokei would resolve the problem.